demos/gaze_estimation_demo/cpp/src/ie_wrapper.cpp

```diff
@@ -1,161 +1,158 @@
 
     for (int i = 0; i < dataSize; ++i) {
         output.push_back(buffer[i]);
     }
 }
 
-const std::map<std::string, std::vector<unsigned long>>& IEWrapper::getInputBlobDimsInfo() const {
-    return inputBlobsDimsInfo;
+const std::map<std::string, ov::Shape>& IEWrapper::getInputTensorDimsInfo() const {
+    return input_tensors_dims_info;
 }
-const std::map<std::string, std::vector<unsigned long>>& IEWrapper::getOutputBlobDimsInfo() const {
-    return outputBlobsDimsInfo;
+const std::map<std::string, ov::Shape>& IEWrapper::getOutputTensorDimsInfo() const {
+    return output_tensors_dims_info;
 }
 
 std::string IEWrapper::expectSingleInput() const {
-    if (inputBlobsDimsInfo.size() != 1) {
+    if (input_tensors_dims_info.size() != 1) {
         throw std::runtime_error(modelPath + ": expected to have 1 input");
     }
 
-    return inputBlobsDimsInfo.begin()->first;
+    return input_tensors_dims_info.begin()->first;
 }
 
 std::string IEWrapper::expectSingleOutput() const {
-    if (outputBlobsDimsInfo.size() != 1) {
+    if (output_tensors_dims_info.size() != 1) {
         throw std::runtime_error(modelPath + ": expected to have 1 output");
     }
 
-    return outputBlobsDimsInfo.begin()->first;
+    return output_tensors_dims_info.begin()->first;
 }
 
-void IEWrapper::expectImageInput(const std::string& blobName) const {
-    const auto& dims = inputBlobsDimsInfo.at(blobName);
+void IEWrapper::expectImageInput(const std::string& tensorName) const {
+    const auto& dims = input_tensors_dims_info.at(tensorName);
 
     if (dims.size() != 4 || dims[0] != 1 || dims[1] != 3) {
-        throw std::runtime_error(modelPath + ": expected \"" + blobName + "\" to have dimensions 1x3xHxW");
+        throw std::runtime_error(modelPath + ": expected \"" + tensorName + "\" to have dimensions 1x3xHxW");
     }
 }
 
 void IEWrapper::infer() {
-    request.Infer();
+    infer_request.infer();
 }
 
-void IEWrapper::reshape(const std::map<std::string, std::vector<unsigned long> >& newBlobsDimsInfo) {
-    if (inputBlobsDimsInfo.size() != newBlobsDimsInfo.size()) {
-        throw std::runtime_error("Mismatch in the number of blobs being reshaped");
+void IEWrapper::reshape(const std::map<std::string, ov::Shape>& newTensorsDimsInfo) {
+    if (input_tensors_dims_info.size() != newTensorsDimsInfo.size()) {
+        throw std::runtime_error("Mismatch in the number of tensors being reshaped");
     }
 
-    auto inputShapes = network.getInputShapes();
-    for (auto it = newBlobsDimsInfo.begin(); it != newBlobsDimsInfo.end(); ++it) {
-        auto blobName = it->first;
-        auto blobDims = it->second;
-
-        InferenceEngine::SizeVector blobDims_(blobDims.data(), blobDims.data() + blobDims.size());
-        inputShapes[blobName] = blobDims_;
+    std::map<std::string, ov::PartialShape> partial_shapes;
+    for (auto it : newTensorsDimsInfo) {
+        partial_shapes[it.first] = it.second;
```

Comment:
Here you fill the `partial_shapes` mapping by `ov::Shape` type, although it should accept `ov::PartialShape`. I'm not sure this type conversion will work for all cases for reshaping. Could you use `get_partial_shape()` function instead of `get_shape()` while setting the mapping for input/output tensors, please?

Reply:
done